rules/detekt/src/test/kotlin/com/twitter/compose/rules/detekt/ComposeViewModelInjectionCheckTest.kt

```diff
@@ -16,7 +16,7 @@ class ComposeViewModelInjectionCheckTest {
     private val rule = ComposeViewModelInjectionCheck(Config.empty)
 
     @ParameterizedTest
-    @ValueSource(strings = ["viewModel", "weaverViewModel", "hiltViewModel"])
+    @ValueSource(strings = ["viewModel", "weaverViewModel", "hiltViewModel", "injectedViewModel", "mavericksViewModel"])
     fun `passes when a weaverViewModel is used as a default param`(viewModel: String) {
```

Comment:
you can probably make use
val vmNames = arrayOf("viewModel", "weaverViewModel", "hiltViewModel", "injectedViewModel", "mavericksViewModel")
if you want to avoid some of this boilerplate in all the tests

Reply:
Yea this is getting old pretty fast